atmo/jobs/models.py

```diff
@@ -226,3 +213,53 @@ def delete(self, *args, **kwargs):
 
     def get_results(self):
         return self.provisioner.results(self.identifier, self.is_public)
+
+
+@python_2_unicode_compatible
```

Comment:
Yeah but they are on every model class. I thought a separate commit could take care of it.